blazar/policies/networks.py

```diff
@@ -53,7 +53,22 @@
                 'method': 'DELETE'
             }
         ]
-    )
+    ),
+    policy.DocumentedRuleDefault(
+        name=POLICY_ROOT % 'get_allocations',
+        check_str=base.RULE_ADMIN,
+        description='Policy rule for List/Get Network(s) Allocations API.',
```

Comment:
```suggestion
        description='Policy rule for List Network(s) Allocations API.',
```

Reply:
I'll keep it consistent with os-hosts convention for now.